doc/availability.md

```diff
@@ -109,17 +109,17 @@ Reference Area UK has now become available, this is because if the user were to
 
 ### Temporal coverage
 
-For DSDs that have a time dimension, temporal coverage can be included in the DataConstraint's `ReferencePeriod` element. The `ReferencePeriod` is used to indicate the earliest and latest observation dates available for the sub-cube of data based on the current data query.  
+For DSDs that have a time dimension, temporal coverage can be included in the AvailabilityConstraint's `validFrom` and `validTo` property that are part of the Cube Region's Key Value. If provided these properties indicate the earliest and latest observation dates available for specific Component values based on the current data query.  
```

Comment:
Do we need a comma after "If provided"? I'm never sure about the rules for French, let alone English 😂...